src/ResourceManager/Network/Commands.Network/ApplicationGateway/FirewallConfiguration/AzureApplicationGatewayWebApplicationFirewallConfigurationBase.cs

```diff
@@ -31,6 +33,24 @@ public class AzureApplicationGatewayWebApplicationFirewallConfigurationBase : Ne
         [ValidateNotNullOrEmpty]
         public string FirewallMode { get; set; }
 
+        [Parameter(
+               Mandatory = true,
+               HelpMessage = "The type of the web application firewall rule set. Possible values are: 'OWASP'.")]
+        [ValidateNotNullOrEmpty]
+        public string RuleSetType { get; set; }
+
+        [Parameter(
+               Mandatory = true,
+               HelpMessage = "The version of the rule set type.")]
+        [ValidateNotNullOrEmpty]
+        public string RuleSetVersion { get; set; }
```

Comment:
@jobatzil adding parameters `RuleSetType` and `RuleSetVersion` that are mandatory is a breaking change - existing scripts that use these cmdlets will not work